Okta.Xamarin/Okta.Xamarin/Views/NewItemPage.xaml.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+using Okta.Xamarin.Models;
+using Okta.Xamarin.ViewModels;
+
+namespace Okta.Xamarin.Views
+{
+    public partial class NewItemPage : ContentPage
```

Comment:
Do we need this class?

Reply:
This class is used by the Xamarin forms shell.  We _may_ choose to refactor it away in the future.